_config.yml

```diff
@@ -1,15 +1,17 @@
 name: ROS2 Design
 description: "Distilled design documents related to the ROS 2.0 effort"
 
-baseurl: http://design.ros2.org
+baseurl: ""
```

Comment:
It's a standard extension point for jekyll: https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/

We could strip it entirely but I don't think it's worth doing that.